modules/ROOT/partials/toolbar-button-ids/toc-toolbar-buttons.adoc

```diff
@@ -1,6 +1,6 @@
 [cols="1,3",options="header"]
```

Comment:
This file needs renaming to `tableofcontents-toolbar-buttons.adoc`.

- [ ] Rename file to match new plugin name/code